tests/failing-symbolic.haskell

```diff
@@ -1,4 +1,3 @@
-tests/specs/benchmarks/ecrecover00-sigvalid-spec.k
 tests/specs/benchmarks/ecrecoverloop00-sig1-invalid-spec.k
```

Comment:
Were there any other tests that started passing with this change?

Reply:
I didn't get exit code 0 for any new tests